### PR TITLE
1st wave of button type fix (#1416)

### PR DESCRIPTION
**Pull request by roderickdevelopers**
I edited 4 files for the button type fix. I will do others after this one is good. I would like to only edit 3 or 4 at a time. @locks maybe add an item to explain if the developer would need to specify the button type.

### CODE REVIEW

File: guides/release/accessibility/components.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Browsers have implemented the spec in a way that provides functionality for free
 For example, consider this code sample: 
 
 ```html
-<button>Submit Form</button>
+<button type="submit">Submit Form</button>
```
Comment (locks) [inline guides/release/accessibility/components.md:9]: Can you revert this change? Since it's in the accessibility section of the guides I'm inclined to keep the current example as it's likely purposeful.

Comment (MelSumner) [reply guides/release/accessibility/components.md:9]: This looks good to me @locks
